cluster/node-pools/worker-ubuntu-default/userdata.yaml

```diff
@@ -52,6 +52,9 @@ write_files:
       featureGates:
         SupportPodPidsLimit: true
       podPidsLimit: {{ .NodePool.ConfigItems.pod_max_pids }}
+{{- end }}
+{{- if eq .NodePool.ConfigItems.cpu_management_policy "static" }}
```

Comment:
Are there more resonable options? If yes it would make sense to do:
```
{{- if ... }}
  cpuManagerPolicy: {{ .NodePool.ConfigItems.cpu_management_policy }}
{{- end }}
```

Reply:
Or even drop the `if` if you have a default.

Reply:
There's only `none` and `static`, but it might make sense to just drop the if since we are going to roll the nodes for v1.14 anyway, so the if doesn't provide so much in this case :)

Reply:
I have dropped the `if` and used the same keyword everywhere.